henriqueApp/app/addUser.tsx

```diff
@@ -56,10 +69,16 @@ export default function AddUserScreen(){
         onBlur={handleBlur('role')}
         errorMessage={touched.role && errors.role ? errors.role : undefined}
       />
-      <Button title="Add User" onPress={() => handleSubmit()} />
+      
+      <CustomButton
+        label="Add User"
+        backgroundColor="#6200ee"
+        textColor="#ffffff"
```

Comment:
Lá na reunião de front vai ficar mais claro, mas o que fazemos no projeto é definirmos um tipo (ou **variante**) de um botão, e isso que seta um conjunto de atributos e estilizações fixos de forma que todo botão do mesmo tipo tenha o mesmo estilo e comportamento.

No fim das contas, cada variante tem também a sua própria função, e isso bom pro usuário "aprender" o que cada botão faz, também, além de manter o código mais consistente porque você não precisa passar props como, por exemplo, essas cores, toda a vez que for utlizar.